llarp/util/logging.hpp

```diff
@@ -25,6 +26,8 @@ namespace llarp
 // Deprecated loggers (in the top-level llarp namespace):
 namespace llarp
 {
+  inline std::shared_ptr<log::RingBufferSink> logRingBuffer = nullptr;
```

Comment:
is there a good reason to have this be a "global"?

Reply:
We talked about this in more depth off github; the answer is that the logging system is inherently (already) using static globals because of how logging needs to done easily from anywhere: needing to carry around an instance to get to the logger makes logging much more painful.  As a result, the sink will already be held in static global variables; this is just storing another pointer to it here because it's a bit awkward to extract it from the main logging system internals.